### ISSUE TITLE
Read if Files Exist or Not, Only Add if They Don't

### ISSUE THREAD

**Issue by una** [Stephn-R/sass-director-sublime#5]
Since this is in Sublime, I think Sass Director could be used *during* a project process, not only at the beginning to set up the files. Presently, if you add a new file to the manifest, it re-creates all of the others and erases any work you've done in those files.

It would be cool if during my project, I could add a new component to the main.scss file, run Sass Director, and that component would be created, preserving the rest of my work as well.

An improvement on the initial concept, taking it from a set-up architecture tool to an integrated production tool :)

**Comment by Stephn-R** [comment]
I like it, this is a really good idea. what about the following:

```Sass Director: Update Manifest```

This command would gather the imports and create any files that don't exist

**Comment by una** [comment]
Yeah. That's fair. So it could be:

`Sass Director: Initial Build of Files`
`Sass Director: Update Files`

And this might be easier to code.

That being said, I feel like it would just make the most sense to simply have one `Sass Director: Build Files` action.

**Comment by Stephn-R** [comment]
Sounds good, I added these as planned features

**Comment by una** [comment]
This is pretty important so I just want to bring it up again :) It would make Sass Director usable more than once in a project's lifespan

**Comment by Stephn-R** [comment]
Yes! That is the plan, you should check out the newest feature for saving Manifest files :)
